packages/block-editor/src/components/block-list/use-block-props/index.js

```diff
@@ -152,6 +152,7 @@ export function useBlockProps( props = {}, { __unstableIsHtml } = {} ) {
 
 	return {
 		tabIndex: blockEditingMode === 'disabled' ? -1 : 0,
+		draggable: true,
```

Comment:
This feels like an impactful change no? Do we expect anything weird?

Reply:
Yes, because I generalised it to all blocks if there is no text selection. I will polish the conditions a bit and test with other blocks, but it would be cool if you can drag any block. I really love that you can now also drag list items from the bullet etc.